src/components/Slider.react.js

```diff
@@ -26,6 +26,7 @@ export default class Slider extends Component {
                 data-dash-is-loading={
                     (loading_state && loading_state.is_loading) || undefined
                 }
+                style={{display: 'initial'}}
```

Comment:
overrides browser default styling `display: block` for default div behavior -- this allows the child to have the correct size

Reply:
🤔 IE11? https://developer.mozilla.org/en-US/docs/Web/CSS/initial#Browser_compatibility

A little fiddling indicates that for `display`, `initial` means `inline`? Can we use that?

Reply:
Oh.. did not realize IE11 did not support `initial`. Thanks for checking it out.

Reply:
Shouldn't the display be inline-block anyway, like here? https://github.com/plotly/dash-core-components/blob/8ccbb40389e5f9eb714ca3b2b1fce87a77ab7fec/src/components/DatePickerRange.react.js#L130

Reply:
So.. realizing that maybe just doing ```style={vertical ? { height: '100%' } : {}}``` on the wrapper div might be just as effective.